package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "start": "yarn run dev",
     "verbose-dev": "DEBUG=verbose,info,error nf start --procfile Procfile.dev",
-    "dev": "DEBUG=info,error nf start --procfile Procfile.dev",
+    "dev": "DEBUG=info,error nf start --procfile Procfile.dev --env .env.dev",
```

Comment:
I think this should just use `.env` (which is not checked into Git and so your local `.env` _is_ your development `.env`).